src/app/components/bookmark-menu/bookmark-menu.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="bookmark-shortcut" >
+  <mat-chip-list>
+    <mat-basic-chip [hidden]="!isBookmarkOpen" *ngFor="let bookmark of topBookmarks;let index = index" class="bookmark-chip bookmark-view" (click)="viewBookmark(bookmark.id)">
+      <span class="bookmark-name">#{{index+1}} - {{ bookmark.name}}</span>
+    </mat-basic-chip>
+    <mat-basic-chip [hidden]="!isBookmarkOpen" class="bookmark-chip bookmark-action-left" style="margin-left:5px" matTooltip="Add a dataset" (click)="openDialogAdd()">
```

Comment:
we can pipe 'Add dataset' with `translate`

Reply:
Done